pkg/config/config.go

```diff
@@ -12,61 +12,46 @@ import (
 	klog "k8s.io/klog/v2"
 )
 
-const (
-	API_SERVER_URL           = "https://kubernetes.default.svc"
-	DEFAULT_CONTEXT_PATH     = "/searchapi"
-	DEFAULT_QUERY_LIMIT      = 10000
-	DEFAULT_QUERY_LOOP_LIMIT = 5000
-	HTTP_PORT                = 4010
-	RBAC_POLL_INTERVAL       = 60000  // 1 minute
-	RBAC_INACTIVITY_TIMEOUT  = 600000 // 10 minutes
-	SERVICEACCT_TOKEN        = ""
-	DEFAULT_DB_PASS          = ""
-	DEFAULT_DB_HOST          = "localhost"
-	DEFAULT_DB_USER          = ""
-	DEFAULT_DB_NAME          = ""
-	DEFAULT_DB_PORT          = int(5432)
-)
```

Comment:
This adds unnecessary complexity.  Defining the default values directly in the code below.